### PR TITLE
Data Picker ver. 1

### PR DESCRIPTION
**Pull request by nikos83**

### CODE REVIEW

File: app/assets/javascripts/datapicker.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+$( document ).on('turbolinks:load', function() {
+    $('#datepicker').datepicker();
```
Comment (fab-ian) [inline app/assets/javascripts/datapicker.js]: If you set 'format' you will be have the possibility to save the record.
```
$('.datepicker').datepicker({
        format: "dd-mm-yyyy"
    });
})
```

File: app/assets/stylesheets/datapicker.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/*!
```
Comment (fab-ian) [inline app/assets/stylesheets/datapicker.scss]: Third-party components add please in "/vendor" folder. The javascript file of this component too.

File: app/assets/javascripts/datapicker.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+$( document ).on('turbolinks:load', function() {
+    $('.datepicker').datepicker({
+        todayHighlight: true,
+        format: "dd-mm-yyyy"
```
Comment (fab-ian) [inline app/assets/javascripts/datapicker.js]: Please add ``` autoclose: true ```. It close the calendar after a value was choiced.

Comment (nikos83) [reply app/assets/javascripts/datapicker.js]: I see. Format is wrong :(

File: app/views/maintenances/_form.html.haml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 = simple_form_for @maintenance do |f|
   = f.input :name, as: :string
   = f.input :mileage, as: :integer
-  = f.input :warranty_date, as: :string, input_html: {class: 'datepicker'}
+  = f.input :warranty_date, as: :string, input_html: { class: 'datepicker', value: localize(f.object.warranty_date, format: '%d-%m-%Y')}
```
Comment (fab-ian) [inline app/views/maintenances/_form.html.haml]: Good idea!
You can shorten it to ``` l(f.object.warranty_date) ``` or ``` l f.object.warranty_date ```
l = localize
The default format for PL is set : %d-%m-%Y. Better don't set it here. Other languages won't be formated correctly.

Comment (nikos83) [reply app/views/maintenances/_form.html.haml]: I think need to be:
  = f.input :warranty_date, as: :string, input_html: { class: 'datepicker', value: (l(f.object.warranty_date, format: '%d-%m-%Y') if f.object.warranty_date.present?) }

Where then should be format included?

**Pull request merged**